mkdocs.yml

```diff
@@ -110,6 +110,7 @@ pages:
       - Monitoring ClickHouse: integration/clickhouse.md
       - Monitoring Docker: integration/docker.md
       - Monitoring Elasticsearch: integration/elasticsearch.md
+      - Monitoring Express.js: integration/express.js.md
```

Comment:
Should we add a card for it to https://sematext.com/docs/integration/ ?
Note that we have Express.js over on https://sematext.com/integrations/ but the info/link to docs in the description should be updated when this gets published.

Reply:
I've added a card to the `docs/integration`. I'll sync with Jorge to update the WWW after I publish this change in Docs.